### PR TITLE
Role card and logic

### PR DESCRIPTION
**Pull request by Schartelb**

### CODE REVIEW

File: app/globals.css [hunk 1]
```diff
@@ -90,8 +90,17 @@ body {
   text-align: center;
   margin: 0px auto 32px auto;
   width: 40%;
+
+  .role{
```
Comment (blinktenor) [inline app/globals.css:94]: if this is empty we should remove it

File: app/globals.css [hunk 2]
```diff
@@ -90,8 +90,17 @@ body {
...
+  .title{
+    font-size:x-large;
+    padding:16px;
+  }
   .ability{
-    width:60%;
-    align-items:center;
+    font-size:large;
   }
+  .unveil{
```
Comment (blinktenor) [inline app/globals.css:104]: do all of the closing } line up? this looks a little wonky, but it might be the git interpreter

Comment (blinktenor) [reply app/globals.css:104]: thats where the build is failing, should be good after that

**Review by blinktenor — APPROVED** [review]
Approved once the css rules are fixed

File: app/globals.css
Comment (Schartelb) [inline app/globals.css:1]: updated

File: pages/game/roleScreen.tsx
Comment (Schartelb) [inline pages/game/roleScreen.tsx:1]: updated

**Review by Schartelb — COMMENTED** [review]
updated as mentioned

**Pull request merged**